Program.cs

```diff
@@ -56,19 +56,18 @@ public static IHostBuilder CreateHostBuilder(string[] args)
 
         private static void CreateDbIfNotExists(IHost host)
         {
-            using (var scope = host.Services.CreateScope())
+            using var scope = host.Services.CreateScope();

```

Comment:
Don't have to change this, since this is out of context from your PR

Reply:
Uh no, I accidentally run my linter.